RPGManager_development/src/rpg_database/character_sheet/Money.java

```diff
@@ -0,0 +1,25 @@
+package rpg_database.character_sheet;
+
+public class Money {
+	int gold;
+	int silver;
+	int copper;
+	int money_temp;
+	public Money(int money){
+		while(money >= 10000){

```

Comment:
splitting the values into gold, silver and copper is unnecessary in the constructor. you just need to return the right place-value